### ISSUE TITLE
Consider switching to right-handed coordinate system

### ISSUE THREAD

**Issue by gherold** [acoular/acoular#373]
For historical reasons, Acoular currently uses a left-handed coordinate system for handling spatial information (grid definitions, microphone array geometries, flow properties, ray tracing, ...).

Since most of the world (including other python packages) by convention uses right-handed coordinates, we should consider making these the default in Acoular as well. This would mean a major compatibility break with older scripts in a lot of cases, but the benefits for Acoular in terms of interoperability and intuitive usage also weigh heavy in my opinion.

We could either

1. completely switch to right-handed coordinates or
2. have a flag that can be set to RH or LH coords in the config

Benefit of 2. would be that this allows easy support for older scripts using Acoular. Benefit of 1. would be keeping a better maintainability of the code base.

**Comment by esarradj** [comment]
In the `environment` module, this affects

- [ ] UniformFlowEnvironment (?)
- [ ] Slot Jet
- [ ] RotatingFlow
- [ ] GeneralFlowEnviroment (?)

In the `tbeamform` module this affects 

- [ ] BeamformerTimeTraj (?)

Interestingly I could not find any other code, but I might be wrong.

**Comment by esarradj** [comment]
There is another situation which might be the basis of all the confusion about Acoular using a left-handed system, which actually not happens inside Acoular, but only when plotting or interpreting the results as e.g. shown [here](https://www.acoular.org/auto_examples/introductory_examples/example_3D_beamforming.html#sphx-glr-auto-examples-introductory-examples-example-3d-beamforming-py). The very same Acoular code could be used with a right handed co-ordinate system. This would mean to define the z-extension of the grid with negative z-values. The only thing that needs to be changed then are the plotting commands! But not Acoular, obviously. Do you agree, @gherold @adku1173 ?

**Comment by gherold** [comment]
Yes, for most (if not all cases), using a left-oriented system now appears like a self-imposed obstacle not required by the original implementation. This is actually a most desirable outcome, since no compatibility-breaking changes are required and everybody can choose whichever system they like better (as long as they use it consistently).

**Comment by esarradj** [comment]
I was under the impression that there would be some change to the cross product when using LHS or RHS. Even this is [not the case](https://stackoverflow.com/questions/4124041/is-opengl-coordinate-system-left-handed-or-right-handed?rq=3).
